temporal/t.upgrade/t.upgrade.html

```diff
@@ -0,0 +1,26 @@
+<h2>DESCRIPTION</h2>
+
+This module upgrades the TGIS DB version from version 2 (DB scheme before
+introducing band reference support ("image collections"), see
+<a href="https://github.com/OSGeo/grass/pull/63">PR #63</a>) to version 3
```

Comment:
Is this such an exemplary PR that you want it linked in the documentation? At this point, I think everybody assumes this would be difference in TGIS between v7 and v8, so refer to 8.0.0.